osquery/core/linux/cpu.h

```diff
@@ -0,0 +1,66 @@
+/**
+ *  Copyright (c) 2018-present, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed as defined on the LICENSE file found in the
+ *  root directory of this source tree.
+ */
+
+#pragma once
+
+#include <osquery/expected.h>
+
+#include <string>
+#include <vector>
```

Comment:
not used?

Reply:
Yes =\ it is supposed to be `bitset`, thanks!